### DISCUSSION TITLE
Alexa media player

### DISCUSSION THREAD

**Discussion by MarcoFranchi** [indomus/forum#245] [HUB: Home Assistant]
Dopo una re-installazione Alexa Media player mi da problemi: l'integrazione la installo senza errori da HACS ma quando vado in impostazioni --> Integrazioni per fare la configurazione mi dice "Impossibile caricare il flusso di configurazione" e mi trovo questo errore nel log
AttributeError: module 'pyotp' has no attribute 'TOTP'

lo sviluppatore [dice](https://github.com/custom-components/alexa_media_player/issues/1265) che c'e' da aggiornare pyopt.
non sapendo bene come fare ho usato ssh & Web Terminal:
verificando con pip noto stranamente che pyopt non è proprio installato e lo installo come consigliato. 
`pip install pyotp~=2.4.1` 
ma il problema persiste.

ho un dubbio e mi servirebbe una mano per capire, non è che alexa madia palyer è installato in un docker e quando mi collego con ssh sono in un ambiente sbagliato? (forse per quello che non vedo pyopt installato)
in linea generale se qualcuno qui è esperto di microservizi, docker e relativa architettura HA e spiega un po il funzionamento potebbe essere molto utile.
poi magari il mio problema non è riconducibile a docker, mah.

nel mio caso tutto è installato su un hardware Odroid come Home Assistant OS standard

grazie 1000
   MArco

**Comment by sky7176** [comment]
Ciao anche a me piacerebbe capire meglio come funziona docker ed affini,
mi associo e attendo.

Giuseppe
